lib/screens/profile_screen.dart

```diff
@@ -103,25 +105,34 @@ class ProfileScreen extends StatelessWidget {
                     children: [
                       Text(
                         user.displayName,
-                        style: TextStyle(
-                          fontSize: 16,
-                        ),
+                        maxLines: 2,
+                        style: Theme.of(context).textTheme.headline1,
                       ),
                     ],
                   ),
                 ),
               ],
             ),
           ),
-          Text(l10n.userListings),
           Padding(
-            padding: const EdgeInsets.only(top: 16.0),
+            padding: const EdgeInsets.only(left: 20.0, top: 10, bottom: 20),
+            child: Text(
+              l10n.userListings,
+              style: Theme.of(context).textTheme.headline2,
+            ),
+          ),
+
+          Padding(
+            padding: const EdgeInsets.symmetric(horizontal: 16.0),
             child: const Divider(height: 0),
           ),
+
+          // Listings
           Expanded(
             child: Padding(
               padding: const EdgeInsets.symmetric(horizontal: spacing),
               child: PaginateFirestore(
+                header: SizedBox(height: 24),
```

Comment:
NIT: You've been using floats for values like these so maybe change this one too